packages/core/src/util/normalizeConfig.js

```diff
@@ -162,6 +166,22 @@ function normalizeServiceName(config) {
   }
 }
 
+/**
+ * @param {InstanaConfig} config
+ */
+function normalizePackageJsonPath(config) {
+  if (config.packageJsonPath == null && process.env['INSTANA_PACKAGE_JSON_PATH']) {
+    config.packageJsonPath = process.env['INSTANA_PACKAGE_JSON_PATH'];
+  }
+  if (config.packageJsonPath != null && typeof config.packageJsonPath !== 'string') {
```

Comment:
Can we simplify the check by `config.packageJsonPath`  instead of `config.packageJsonPath != null`